File: src/integTest/noScala/app/src/androidTest/java/dev/kibet/android/libproject/app/Lib1JavaTest.java
package dev.kibet.android.libproject.app;

import junit.framework.TestCase;

import dev.kibet.android.libproject.lib1.Lib1Java;

public class Lib1JavaTest extends TestCase {
    public void test1() {
        assertEquals("Lib1Java", new Lib1Java().getName());
    }
}

File: src/integTest/useScalaOnlyTest/app/src/androidTest/scala/dev/kibet/android/libproject/app/Lib1JavaTest.scala
package dev.kibet.android.libproject.app

import junit.framework.{TestCase, Assert}
import dev.kibet.android.libproject.lib1.Lib1Java

class Lib1JavaTest extends TestCase {
  def test1() {
    Assert.assertEquals("Lib1Java", new Lib1Java().getName)
    Assert.assertEquals(List("Lib1Java").last, Option(new Lib1Java().getName).get)
  }
}
